tests/test_hub.py

```diff
@@ -397,3 +411,39 @@ def test_autocare_dlt_classification(classification_dataset: Dataset, tmpdir: Pa
     )
 
     _total(hub, dataset, image_size)
+
+
+def test_autocare_dlt_text_recognition(text_recognition_dataset: Dataset, tmpdir: Path):
+    image_size = 32
+    dataset = text_recognition_dataset
+
+    # temporal solution
+    super_cat = [[c.supercategory, c.name] for c in dataset.categories.values()]
+    super_cat_dict = {}
+    for super_cat, cat in super_cat:
+        if super_cat not in super_cat_dict:
+            super_cat_dict[super_cat] = []
+        super_cat_dict[super_cat].append(cat)
+    super_cat_dict_list = []
+
+    for super_cat, cat in super_cat_dict.items():
+        super_cat_dict_list.append({super_cat: cat})
```

Comment:
I think this will be done in dlt hub.
Do we need this code here?

Reply:
> I think this will be done in dlt hub. Do we need this code here?
 I'll check it